src/components/pane.css

```diff
@@ -41,9 +41,27 @@
 				align-items: center;
 			}
 
-			& .ck-inspector-tree__config label {
-				margin-left: 1em;
-				margin-right: 1em;
+			& .ck-inspector-tree__config {
+				& + .ck-inspector-tree__config {
+					&::before {
+						content: "";
+						border-right: 1px solid #999;
```

Comment:
Wouldn't you like it to have some `var( )` definition use for `#999` config section border color?

Reply:
You're absolute right. Nice catch.   
  
Fixed in 36007bc.